doc/Client-Portal Protocol.md

```diff
@@ -542,24 +556,124 @@ This a format for client reconnects.
        sdpMLineIndex: number(mLineIndex),   // optional in RemovedCandidatesMessage
        candidate: string(candidateSdp)
       }
+
+    object(TransportOptions)::
+      {
+        type: "webrtc" | "quic-p2p",
+        id: string(transportId) | null,  // null will result to create a new transport channel.
```

Comment:
What if a publication has "transport" with type "webrtc" and not-null id?
Is "webrtc"'s TransportOptions' id always null?

Reply:
It could be non-null. That's my though about reusing `PeerConnection` for multiple streams. Actually I haven't figured out what does `transportId` mean for WebRTC connections. Is it a PeerConnection ID? or transceiver ID? For now, `PeerConnection` doesn't have an ID, and transceiver cannot be shared by two tracks in the same direction. But it's quite clear for QUIC connections.

Reply:
May give PeerConnection an ID later

Reply:
If we don't support setting transportId for "webrtc" type, add a comment so people won't try.

Reply:
Done.